minNumPointsToSend: 100
useRANSAC: false

# RANSAC parameters
distanceThreshold: 0.1 # 0.01
probability: 0.99  # NOT IN USE
maxIterations: 1000  # NOT IN USE

# RadiusOutlierRemoval settings
useFiltering: true
activeMinNeighborsInRadius: 4 # 8
activeRadiusSearch: 0.2 # 0.2
marginMinNeighborsInRadius: 4
marginRadiusSearch: 0.2
# StatisticalOutlierRemoval settings
meanK: 10
stddevMulThresh: 0.01

# Clustering parameters
clusterTolerance: 0.2 # 0.02
minClusterSize: 35
maxClusterSize: 5000

# For localization
useReferenceCloud: false
referenceCloudPath: "/home/wfram/.ros/dmvio_global_cloud.pcd"